OpenFusion.vcxproj

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this from a fork? This shouldn't be pushed upstream. [3]

Reply:
fork